html-report/src/components/LogContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import LogFilter from './LogFilter'
+
+export default class LogContainer extends Component {
+
+  state = {
+    logs: [],
+    results: [],
+    loading: true,
+    hide: false
+  }
+
+  componentDidMount() {
+    document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
It has no sense adding listener `DOMContentLoaded` as the method `componentDidMount` itself says that the DOM is ready. Please read the React documentation to understand React way: https://reactjs.org/docs/state-and-lifecycle.html